.github/workflows/test.yml

```diff
@@ -8,9 +8,17 @@ jobs:
     runs-on: ubuntu-16.04
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-\
     - uses: ./
       env:
         JEKYLL_PAT: ${{ secrets.JEKYLL_PAT }}
+      with:
+        jekyll_src: 'sample_site'
```

Comment:
The source directory is picked up automatically based on the presence of _config.yml so not sure why we should add this optional parameters. 

https://github.com/helaili/jekyll-action/blob/5593772ac749b341bfdaf480dc413c11d287f18f/entrypoint.sh#L17

Reply:
Hey, this line prevents this error (https://github.com/limjh16/jekyll-action/runs/603853082#step:4:172) since the find command somehow found a _config.yml in vendor/bundle first. A better way would be to exclude the vendor/bundle folder but I wasn't sure how to do that